### DISCUSSION TITLE
New Wiring Component

### DISCUSSION THREAD

**Discussion by PiatraAlatyr** [FakeFishGames/Barotrauma#10889] [Ideas and suggestions]
The game is missing a very simple but important component - the impulse circuit. Many components emit a signal constantly, but single pulses are often not enough to work with automation.
It should, upon receipt of any signal at the input, produce a similar single pulse at the output (like a button). Convert the constant 1 into a single pulse 1, and so on.

**Comment by mygamingaccount** [comment]
An edge detector you say? 🤔

**Comment by PiatraAlatyr** [comment]
Not really. This component, in fact, should receive a continuous signal to output a single pulse.

**Reply by mygamingaccount** [reply]
So an edge detector

Like #6470
